orchagent/mplsrouteorch.cpp

```diff
@@ -409,9 +460,12 @@ bool RouteOrch::addLabelRoute(LabelRouteBulkContext& ctx, const NextHopGroupKey
     sai_object_id_t& vrf_id = ctx.vrf_id;
     Label& label = ctx.label;
 
+    SWSS_LOG_NOTICE("Adding route for label %u with next hop(s) %s",
```

Comment:
default level of logging includes "NOTICE". This can quickly fill up the logs. There is no similar log for routes.

Reply:
Removed